demisto_sdk/dev_tools/linter.py

```
@@ -79,7 +79,9 @@ def remove_common_server_python(self):
 
     def run_dev_packages(self) -> int:
         # load yaml
-        yml_path = glob.glob(self.project_dir + '/*.yml')[0]
+        _, yml_path = get_yml_paths_in_dir(self.project_dir, Errors.no_yml_file(self.project_dir))
+        if not yml_path:
+            return 1
```

I think we need to raise an exception here